arm/Microsoft.Sql/servers/.parameters/admin.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-az-sqlsrv-admin-001"
+        },
+        "administrators": {
+            "value": {
+                "azureADOnlyAuthentication": false,
```

Comment:
"azureADOnlyAuthentication" should be `true` as no Login credential is provided.

Reply:
testing